multinet/auth/google.py

```diff
@@ -136,7 +146,15 @@ def authorized(state: str, code: str) -> ResponseWrapper:
     user.save()
 
     return_url = session.pop("return_url", default_return_url())
-    resp = make_response(redirect(ensure_external_url(return_url)))
-    session[MULTINET_COOKIE] = cookie
-
+    resp: Response = make_response(redirect(ensure_external_url(return_url)))
+
+    # If running in development, don't set secure and samesite cookie attributes
+    development = current_app.config.get("ENV") == "development"
```

Comment:
Instead of computing this bool here, I wonder if we could do it at app startup time, or maybe move it to a util function?

It's not critical to change this now, but I'd at least like to get an issue filed in that case.